build.gradle

```diff
@@ -22,14 +22,20 @@ plugins {
 configurations {
     resultJar
     emfJarConfig
+    aspectJarConfig
 }
 
 dependencies {
     resultJar project(path: ':kieker-tools', configuration: 'resultJar')
     resultJar 'org.jctools:jctools-core:2.1.0'
     resultJar 'org.slf4j:slf4j-api:1.7.25'
 
-    emfJarConfig project(path: ':kieker-analysis', configuration: 'emfJarConfig')
+    //emfJarConfig project(path: ':kieker-analysis', configuration: 'emfJarConfig')
```

Comment:
I would recommend to remove unused code completely (and not just leave it as a comment), since it is still contained in the version control system. This is a general pattern in the buildfiles, maybe we should discuss this at a meeting.